kubelet/datadog_checks/kubelet/prometheus.py

```diff
@@ -290,18 +293,26 @@ def _process_container_metric(self, type, metric_name, metric, scraper_config):
                 tags += self._get_kube_container_name(sample[self.SAMPLE_LABELS])
                 tags = list(set(tags))
 
+            for label in labels:
+                value = sample[self.SAMPLE_LABELS][label]
```

Comment:
you should probably `get` here `label` might not be in the dict

Reply:
makes sense, thanks!

Reply:
done